src/ocean/text/convert/Formatter.d

```diff
@@ -86,7 +86,7 @@ public istring format (Args...) (cstring fmt, Args args)
 
 *******************************************************************************/
 
-public mstring sformat (Args...) (ref mstring buffer, cstring fmt, Args args)
+public mstring aformat (Args...) (ref mstring buffer, cstring fmt, Args args)
```

Comment:
Note, the original equivalent in `ocean.text.convert.Layout_tango` was `Layout.format`, so we could go back to using the `format` name instead of introducing this new one.